openwrt.md

```diff
@@ -16,21 +16,23 @@ OpenWRT 是一种嵌入式 Linux 操作系统，广泛应用于家用路由器/
 虽然 IETF 的设计中，IPv6 不再有 NAT (网络地址转换), 但 Linux 内核从 3.7 版本开始实现了 IPv6 的 NAT。早年也曾有过 NAT66 等非官方项目，但缺乏维护，目前已经不再推荐使用。
 OpenWRT IPv6 NAT 配置部分，由 [@Blaok](https://blog.blaok.me/) 贡献。
 
-### 零: 安装内核模块和有用的软件包
+### 零: 检查内核模块和有用的软件包
 
 ```
-opkg install ip kmod-ipt-nat6 kmod-ip6tables luci-ipv6 iputils-traceroute6
+ip kmod-ipt-nat6 kmod-ip6tables luci-ipv6 iputils-traceroute6
 ```
 
+`kmod`开头的内核模块一般无法通过opkg直接安装，推荐在编译固件时就将这些软件包放入固件
+
```

Comment:
也不完全是， 如果是用固定的Release版本，
比如安装的是15.05版本提供的img，编译kmod时，使用对应的git branch来进行编译，
编出来的kmod是可以被正常insert的

但是用了Trunk的话就比较糟糕，你编译kmod的时候要找到对应的那一次commit，
在这下面编译才行，否则装载kmod会报kernel version dependency错误。
但这时候你强制load进来也没啥问题。除非代码差的特别多。

编译的时候把对应的kmod选成'M'状态就可以了。

On Mon, 21 Mar 2016, Blaok wrote:

>>  ```
>>
>> +`kmod`开头的内核模块一般无法通过opkg直接安装，推荐在编译固件时就将这些软件包放入固件
>> +
>
> 好的吧没有说太清楚，内核模块一般是完全没法安装，其他的也最好不要opkg安装，因为opkg安装软件包非常坑爹，会额外占用很多空间，最好都是在编译固件时都放进去。我再改一下
>
> ---
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/tuna/ipv6.tsinghua.edu.cn/pull/11/files/3cc0b06c34fc08f8e9799e4ac88eeb681ef423bb#r56943705